### ISSUE TITLE
Create a minimalistic UI client using the chat-e2ee/service SDK

### ISSUE THREAD

**Issue by muke1908** [muke1908/chat-e2ee#297]
Use the SDK @chat-e2ee/service and build a minimalistic UI
https://github.com/muke1908/chat-e2ee/tree/master/service

Scope of this task:
 - Create a separate folder for the client.
 - Create the project (typescript + any UI framework)
 - Install the SDK  and initialize it.

**Comment by Jeetjyoti-Deka** [comment]
Hey, Mukesh just came across your project and this issue.
I am at a very beginner level for open source but I would like to contribute in this issue.
So I just wanted to make sure that what you want to do in this issue is change the app's frontend ?

**Comment by EmilioBerumen910** [comment]
I would like to work on this issue

**Comment by muke1908** [comment]
@EmilioBerumen910 Sure, please go ahead.

**Comment by EmilioBerumen910** [comment]
@muke1908 Thank you very much for the opportunity to participate in this project. I wanted to share my concept for how the minimalist client UI might appear and would appreciate any feedback before beginning the implementation process. This are some drafts of how I visualize the project UI: 
![1](https://github.com/muke1908/chat-e2ee/assets/133551592/a7541621-bcdb-4c38-b7c8-5285c41d3648)
![2](https://github.com/muke1908/chat-e2ee/assets/133551592/b057bd55-20ca-4f1a-8208-189c9c4ff7bb)
![3](https://github.com/muke1908/chat-e2ee/assets/133551592/d0d85c7a-196d-4d00-bc9d-51abedb26bb6)

**Comment by muke1908** [comment]
Looks great! 
Some suggestions: 
1 - let's keep the primary color (which is orange here) configurable in the code. 
2 - It can display the QR code along with the generated link - no separate button needed.

**Comment by lumariaap15** [comment]
Hey @muke1908 why is this issue still open? I'd like to contribute on your project! Is this a good place to start?

**Comment by muke1908** [comment]
@lumariaap15 sure, please go ahead.

**Comment by Abirami04M** [comment]
@muke1908  Is the issue still open? Can I contribute to it?

**Comment by muke1908** [comment]
@Abirami04M , yes. Just make sure that you create a separate folder for your UI code. It needs to be unique so that it doesn't conflict with others (if any). 
Probably `./ui-abirami04M/` ? 

@lumariaap15 , if you are also making an UI, can you also please make sure that it's in a unique folder like `'./ui-lumariaap15` ?

**Comment by Abirami04M** [comment]
Ok, just to clarify, can I use React for the UI?  and is there any time
span within which i should complete the work?

**Comment by muke1908** [comment]
Sure, feel free to use any framework.
